yarn.lock

```diff
@@ -4469,6 +4469,37 @@ __metadata:
   languageName: node
   linkType: hard
 
+"azero.dev@workspace:.":
```

Comment:
package.json changes triggered this change? :thinking:

Reply:
Correct